Toy Update

Fikou opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request

<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->
Adds action figures of the prisoner and psychologist role
Adds missing action figures to Delta and Kilo, moves some around
Fixes Intento high score system not working well
Resprites all mech figures, repaths them, adds ones for mechs that are missing

## Why It's Good For The Game

<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->
its an epic win

## Changelog
:cl:
expansion: new mech action figures and job action figures
fix: Intento leaderboard works good
imageadd: new sprites for mech action figures
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

Fikou committed:
code

Fikou committed:
sprite part 1

Fikou committed:
sprites and mapping

Fikou committed:
wa

Fikou committed:
fix

Fikou committed:
warden figure fix

Fikou committed:
warden fix 2

coiax commented on code/game/objects/items/toys.dm:
```diff
@@ -1321,7 +1320,7 @@
 /obj/item/toy/figure/chef
 	name = "Chef action figure"
```
```suggestion
	name = "\improper Chef action figure"
```

Do this for the others.

coiax commented on code/modules/holiday/easter.dm:
```diff
@@ -158,37 +158,26 @@
 	icon_state = "egg-[eggcolor]"
 
 /obj/item/food/egg/proc/dispensePrize(turf/where)
```
How do you feel about just making this the arcade toy drop list, plus some additional bunny related items? Just a though, not sure if it's a good idea just for refactoring's sake.

Fikou replied:
i think its kind of out of scope for this, not sure

coiax commented on code/game/objects/items/toy_mechs.dm:
```diff
@@ -11,14 +11,19 @@
 	verb_exclaim = "beeps"
 	verb_yell = "beeps"
 	w_class = WEIGHT_CLASS_SMALL
+	/// Number of the toy, used in it's desc
+	var/series_number = 0
```
If the series numbers actual value is unimportant, would it be possible to do something like.

(pseduocode)
```
mech_types = subtypesof(/obj/item/toy/mecha)
series_number = mech_types.IndexOf(src.type)
```

Then you wouldn't need to specify it in each type, or have the ALL_MECH_TOYS define?

Fikou replied:
i tried to number and order it in a specific way and i dont know if that would work with subtypesof

coiax replied:
Ah, if the numbering is important, then keep it.

tralezab replied:
i will pr the element that does it

Fikou committed:
wahwah

Fikou commented on code/game/objects/items/toys.dm:
```diff
@@ -1267,195 +1266,205 @@
 
 /obj/item/toy/figure/borg
-	name = "Cyborg action figure"
+	name = "C\improper yborg action figure"
```
```suggestion
	name = "\improper Cyborg action figure"
```

Fikou committed:
Update code/game/objects/items/toys.dm

maxymax13 commented:
janitor is now soulless 0/10

tralezab committed:
series

Fikou committed:
merge

https://cdn.discordapp.com/attachments/390728717682606081/856476021674737684/unknown.png

Krysonism reviewed (approved):
looks ok!
